<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main</class>
 <widget class="QMainWindow" name="Main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#titulo {
	background-color: rgb(0, 0, 0);
}

QLabel#lbl_titulo {
	color: rgb(255, 255, 255);
    font-size: 20px;
}

QLabel, QLineEdit, QPushButton {
	font-size: 14px;
}

QPushButton#btn_salvar {	
	background-color: rgb(0, 170, 0);
	color: rgb(255, 255, 255);
    border: none;
}

QPushButton#btn_fechar {	
	background-color: rgb(170, 0, 0);
	color: rgb(255, 255, 255);
    border: none;
}
</string>
  </property>
  <widget class="QWidget" name="painel_principal">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QWidget" name="titulo" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="lbl_titulo">
         <property name="text">
          <string>CADASTRO E LISTAGEM DE FUNCIONÁRIOS</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="componentes" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="lbl_mensagem">
         <property name="styleSheet">
          <string notr="true">color: green;</string>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Nome</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edt_nome">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Sexo</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cmb_sexo">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>Masculino</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Feminino</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Salário</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edt_salario">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Departamento</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edt_departamento">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="botoes" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="btn_fechar">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>Fechar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_salvar">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>Salvar</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="tbl_funcionarios">
      <column>
       <property name="text">
        <string>Nome</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Sexo</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Salário</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Departamento</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
